packages/bierzo-wallet/src/communication/rpcEndpoint.ts

```diff
@@ -15,11 +15,14 @@ export type GetIdentitiesResponse = readonly Identity[];
  */
 export type SignAndPostResponse = TransactionId | null;
 
+export type RpcEndpointType = "ledger" | "extension";
+
 export interface RpcEndpoint {
   readonly authorizeGetIdentitiesMessage: string;
   readonly authorizeSignAndPostMessage: string;
   readonly notAvailableMessage: string;
   readonly noMatchingIdentityMessage: string;
+  readonly rpcEndpointType: RpcEndpointType;
```

Comment:
👍

Reply:
Can we rename this to just `type`, since it is a property of the RPC endpoint anyways. Then we use it like `rpcEndpoint.type` instead of `rpcEndpoint.rpcEndpointType`.